modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -591,96 +622,25 @@ private CompletableFuture<Void> triggerTxRecovery(UUID txId, String senderId) {
     }
 
     /**
-     * Validates that the table exists at a timestamp corresponding to the request operation.
+     * Return tx operation timestamp.
```

Comment:
```suggestion
     * Returns tx operation timestamp.
```

Reply:
BTW, don't we need the detailed breakdown which timestamp is returned for which request type? This information was in the javadoc, but now it's removed.